website/docs/r/is_instance_network_interface.html.markdown

```diff
@@ -23,6 +23,8 @@ Create, update, or delete an instance network interface on VPC. For more informa
   }
   ```
 
+  -> **Allowed vNIC per profile.** 
```

Comment:
is the placing is correct? have a work with Sumit or Astha once.. Otherwise everything looks good to me..

Thank you

Reply:
Yes, the placement on the top of the resource doc is right here, verified with Astha.